Cargo.toml

```diff
@@ -115,8 +115,8 @@ move-ir-types = { git = "https://github.com/move-language/move", rev = "0800fc79
 move-prover = { git = "https://github.com/move-language/move", rev = "0800fc79a98ca304bd449878f545cdcaff6f94bb" }
 move-prover-boogie-backend = { git = "https://github.com/move-language/move", rev = "0800fc79a98ca304bd449878f545cdcaff6f94bb" }
 
-fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "12c522bea01e4a3d19ecae63d789edcdcce2d66c" }
-fastcrypto-zkp = { git = "https://github.com/MystenLabs/fastcrypto", rev = "12c522bea01e4a3d19ecae63d789edcdcce2d66c", package = "fastcrypto-zkp" }
+fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "b6411e014255c9e2a0f6dff00478bc45d76e9f5d" }
```

Comment:
It is ok to have different dep versions of fastcrypto across our Sui codebase? Or when we change in one place we should change everywhere?

Reply:
b6411e is the only version used in Sui. all crates refer to this workspace `fastcrypto = { workspace = true }` cargo to derive version